indymeet/settings/base.py

```diff
@@ -211,7 +211,7 @@
 
 AUTH_USER_MODEL = "accounts.CustomUser"
 
-LOGIN_REDIRECT_URL = "/"
+#LOGIN_REDIRECT_URL = "/"
```

Comment:
I believe this is an expected Django setting, so I'm not sure we want to remove it. Maybe it'd be best to go to the account/profile edit url as a fallback.

Reply:
I was not sure what to do in this scenario. In the documentation I read that the default value for `LOGIN_REDIRECT_URL` is of course `account/profile` but maybe it's better explicit the url? Do you think I just could delete the line and leave the [defaul value](https://docs.djangoproject.com/en/5.0/ref/settings/#login-redirect-url) instead?

Reply:
Sorry for the incredibly late reply. We do have a profile page, let's use the URL for that.

Reply:
It should be accounts/profile to point to this path https://github.com/djangonaut-space/wagtail-indymeet/blob/develop/accounts%2Furls.py#L14

Reply:
To use the default value (the profile page) I will simply delete the variable. Do you agree?

Reply:
Yes, you're correct. I wasn't sure if the default value was `account/profile` or `accounts/profile`.

Reply:
Ok, I've done it. Could you check one last time @tim-schilling ?